app/src/main/java/org/oppia/android/app/help/HelpActivityPresenter.kt

```diff
@@ -16,11 +16,13 @@ class HelpActivityPresenter @Inject constructor(private val activity: AppCompatA
   private lateinit var toolbar: Toolbar
 
   fun handleOnCreate(isFromNavigationDrawer: Boolean) {
-    activity.setContentView(R.layout.help_activity)
-    setUpToolbar()
     if (isFromNavigationDrawer) {
+      activity.setContentView(R.layout.help_activity)
+      setUpToolbar()
       setUpNavigationDrawer()
     } else {
+      activity.setContentView(R.layout.help_without_drawer_activity)
```

Comment:
@anandwana001 this is the line that was commented while testing
as there can not layout set the `doesnotexist()`  was not working.